src/showcase/src/layouts/Screen.astro

```diff
@@ -0,0 +1,11 @@
+---
+import Layout from "../layouts/Layout.astro";
+
+const { title, pageName } = Astro.props;
```

Comment:
do you know if Astro types the props if you don't specify the types? or does is default to `any`?

Reply:
Good point! Yes, I found it here: https://docs.astro.build/en/guides/typescript/#component-props

Done

Reply:
Thanks! As far as I'm concerned you can even drop the `children` field

Reply:
The recommendation is that if the component requires a "slot" we should add it in the props.

This way I got an error when I used `<Screen>` without a child.